kernel/content/ezcontentoperationcollection.php

```diff
@@ -809,7 +809,11 @@ static public function addAssignment( $nodeID, $objectID, $selectedNodeIDArray )
         }
         if ( $locationAdded )
         {
-
+            eZAudit::writeAudit( 'location-assign', array( 'Selected Node ID Array' => implode( ', ' , $selectedNodeIDArray ),
+                                              'Node ID' => $nodeID,
```

Comment:
What node ID is this? Can we maybe provide info on the main node ID and then info about what locations were added?